kano_world/functions.py

```diff
@@ -243,3 +243,16 @@ def mark_notification_read(n_id):
     )
 
     return success, text
+
+
+def get_stats_activity():
+    # Instantiate the global session variable
+    login_using_token()
+    
+    if not glob_session:
```

Comment:
Since `login_using_token` has a return value.  As setting the glob_session is a side effect from the `login_using_token` having it together with the return value of the `login_using_token` would be preferable. Adding logging at `info` level in case of failure could be helpful